src/Compatibility/ControlGallery/src/Core/ControlGalleryPages/CellForceUpdateSizeGalleryPage.cs

```diff
@@ -131,7 +132,7 @@ public MyPage(ListViewCachingStrategy strategy)
 					{
 						var target = _DataSource[i];
 
-						if (Device.RuntimePlatform == Device.iOS)
+						if (DeviceInfo.Platform == DevicePlatform.iOS)
```

Comment:
Should checks like these use the `OperatingSystem` class?

https://docs.microsoft.com/dotnet/api/system.operatingsystem.isios?view=net-6.0

Do we even need `DeviceInfo.Platform`?

Reply:
Is it possible for the BCL API to have a more generic API? Instead of IsXxx and have a string based one? Or even mark some api as a "this method returns true on platform X"? So we can add our own platforms and then inform the linker?

Reply:
I suppose `IsOSPlatform` exists and is linker-friendly? Maybe we can use that?